typescript/sdk/src/ism/HyperlaneIsmFactory.ts

```diff
@@ -116,16 +144,15 @@ export class HyperlaneIsmFactory extends HyperlaneApp<IsmFactoryFactories> {
     );
     await routingIsm.transferOwnership(config.owner);
     const address = dispatchLogs[0].args['module'];
-    return DomainRoutingIsm__factory.connect(address, signer);
+    return IRoutingIsm__factory.connect(address, signer);
```

Comment:
I don't think this change is right, we rely on the specific implementation for the deployer

Reply:
uhhh we do? where

Reply:
Ah okay nvm you're right, we do but we rely on it above, not after the return